db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/task/FailureHandler.java

```diff
@@ -90,10 +93,9 @@ public void onFailure(
       int totalNumberOfFailures = consecutiveFailures + 1;
       if (totalNumberOfFailures > maxRetries) {
         LOG.error(
-            "Execution has failed {} times for task instance {}. Cancelling execution.",
+            "Execution has failed {} times for task instance {}. Invoking maxRetriesExceededHandler",
             totalNumberOfFailures,
             executionComplete.getExecution().taskInstance);
-        executionOperations.stop();
         maxRetriesExceededHandler.accept(executionComplete, executionOperations);
```

Comment:
I am unsure about the original intent of the `maxRetriesExceededHandler`. The handler should not have taken `executionOperations` as a parameter since the execution had already been stopped.